soh/soh/Enhancements/tts/tts.cpp

```diff
@@ -1148,6 +1146,44 @@ void RegisterOnSetGameLanguageHook() {
     GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSetGameLanguage>([]() { InitTTSBank(); });
 }
 
+void RegisterOnSetDoAction() {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSetDoAction>([](uint16_t action) {
+        if (CVarGetInteger(CVAR_SETTING("A11yTTS"), 0)) {
+            uint8_t language = CVarGetInteger(CVAR_SETTING("Languages"), 0);
+            const char* text;
+            switch (action) {
+                case DO_ACTION_CHECK:
+                    text = language == LANGUAGE_FRA ? "voir" : language == LANGUAGE_GER ? "lesen" : "check";
+                    break;
+                case DO_ACTION_ENTER:
+                    text = language == LANGUAGE_FRA ? "enter" : language == LANGUAGE_GER ? "kriechen" : "ranger";
```

Comment:
`enter` -> `entrer`
`ranger` -> `enter`

but in another note, the "putaway" action is missing completely from this switch case, which is where the `ranger` should be, I assume?

Reply:
right, I had removed putaway being read since feedback was that it was too noisy